main/ui/src/main/java/org/cryptomator/ui/unlock/UnlockWorkflow.java

```diff
@@ -129,8 +129,12 @@ private void handleSuccess() {
 				window.setScene(successScene.get());
 				window.show();
 			});
-			case REVEAL -> vaultService.reveal(vault);
-			case IGNORE -> {}
+			case REVEAL -> {
+				Platform.runLater(window::close);
+
+				vaultService.reveal(vault);
+			}
+			case IGNORE -> { Platform.runLater(window::close); }
```

Comment:
no `{}` needed for one-liners

Reply:
To be fair: for one-liners in if/else statements I prefer to _always_ use braces. But in the examples for JEP 361 this isn't the case, so I guess this might become a common practice in this case :D